data/human/syndicate jobs.txt

```diff
@@ -529,3 +529,166 @@ mission "Return Syndicate Prisoners [2]"
 	on complete
 		payment 5000 100
 		dialog "The labor organizers are almost skeletally thin from their imprisonment. One of them raises his face to the sky as though showering in sunlight. You receive your payment of <payment>."
+
+
+mission "Syndicated Shipyards Shipment [0]"
+	name "Shipment to <planet>"
+	description "Syndicated Shipyards, one of the largest shipyards in human space, requests an itinerant captain to deliver <cargo> to <destination> to ensure sufficient stocks. Payment is <payment>."
+	cargo "weapons" 5 2 .1
+	job
+	repeat
+	to offer
+		random < 10
+	source
+		near Achernar 2 10
+		attributes core factory
+	destination Foundry
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 0 160
+		dialog phrase "generic cargo delivery payment"
+
+mission "Syndicated Shipyards Shipment [1]"
+	name "Shipment to <planet>"
+	description "Syndicated Shipyards, one of the largest shipyards in human space, requests an itinerant captain to deliver <cargo> to <destination> to ensure sufficient stocks. Payment is <payment>."
+	cargo "ship alloys" 5 2 .1
+	job
+	repeat
+	to offer
+		random < 10
+	source
+		near Achernar 2 10
+		attributes core mining
+	destination Foundry
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 0 160
+		dialog phrase "generic cargo delivery payment"
+
+mission "Syndicated Shipyards Shipment [2]"
+	name "Shipment to <planet>"
+	description "Syndicated Shipyards, one of the largest shipyards in human space, requests an itinerant captain to deliver <cargo> to <destination> to ensure sufficient stocks. Payment is <payment>."
```

Comment:
‘Itinerant’ doesn’t feel right here. How about ‘dynamic’, ‘industrious’, or ‘reliable’?

Reply:
'itinerant' means wandering, but I could add something like 'industrious' to the description.

Reply:
Yeah, that’s why it didn’t feel quite right. I mean, why are they specifying a wandering captain in the advert? Is there any other type?

Reply:
Maybe those who only ship luxury/food from Dirt Belt to Earth/Paradise worlds.

The job is about extra slack in the Syndicate's production (which isn't already being picked up by their own logistics division) being rerouted by indepedent traders. I guess 'indepedent' might work better.

Reply:
Independent would be good.
Itinerant sounds like you need to give them a schedule of where you're going when.